src/components/action_buttons/PatientsButtonsRenderer.jsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Box } from "@mui/material";
+import DeleteIcon from "@mui/icons-material/Delete";
+import NoteAltIcon from "@mui/icons-material/NoteAlt";
+import InventoryIcon from "@mui/icons-material/Inventory";
+import VisibilityIcon from "@mui/icons-material/Visibility";
+function PatientsButtonsRenderer() {
```

Comment:
We should stay consistent with the way that we structure our components. Preference is to use the const syntax.